src/server/migrations/0.2.0-0.3.0/index.js

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = {
```

Comment:
You should replace this with a template file if you want to include anything. This file looks like it has a real, working migration at first glance.

Reply:
John and I were thinking of writing an actual migration on the next PR and then people will look at that as a sample instead of writing a template file